src/logger.ts

```diff
@@ -30,20 +30,15 @@ function loggingLevelApplies(messageVerbosity: string | undefined): boolean {
 }
 
 //TODO: implement more verbosity levels for the output log
-export function message(message: string, loggingLevel?: string): void {
+export function message(message: string, loggingLevel?: string, show: boolean = true): void {
```

Comment:
My recommendation here is to make "show" be event-based instead of message based.  For example, you show the window when you begin a build.  Or you show the window when the output channel is first created.  Then you leave it to the user to decide when they want to see the output.

Reply:
Yes, good idea. Will do in a separate PR.